scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -556,15 +556,15 @@ class FossIdTest : WordSpec({
             val scanCode = scanCode(PROJECT, FossId.DeltaTag.DELTA)
             val config = createConfig(deltaScanLimit = deltaScanLimit)
             val vcsInfo = createVcsInfo()
-            val originScan = createScan(vcsInfo.url, vcsInfo.revision, originCode, scanId = "${SCAN_ID}0")
+            val originScan = createScan(vcsInfo.url, vcsInfo.revision, originCode, scanId = SCAN_ID)
```

Comment:
This slightly changes the logic in the test, since `originScan` and `otherScan1` now have the same `scanId`. Not sure whether this is a problem, but this change was probably not intendet.

Reply:
This is a valid point.